### ISSUE TITLE
Faster enginePins.stopPins()

### ISSUE THREAD

**Issue by rusefillc** [rusefi/rusefi#3070]
at the moment we can half coils/injectors stuck open for up to half a second on loss of trigger input. in reality that usually comes with relays going off but still maybe we shall detect this condition faster for any trigger or at least consider trigger tooth count in the delay calculation

```
	if (secondsSinceTriggerEvent < 0.5f) {
		// Engine moved recently, no need to safe pins.
		return;
	}

```

**Comment by rusefillc** [comment]
@mck1117 please upload video if feels appropriate

**Comment by rusefillc** [comment]
```
Alex Miculescu  9:45 AM
I was doing timing measurements and left ign on and killed tge engine from ts





9:45
3rd start it was running on 2 cyl
9:47
So probably that killed it
9:47
Because when it runs, it is cool and working
```

TODO @rusefillc @mck1117 to check how is that specific scenario handled

**Comment by mck1117** [comment]
I feel like the engine stop button should work via limp manager, instead of doing whatever crazy logic (force killing the pins??) it has now

**Comment by mck1117** [comment]
will be fixed by #3180

**Comment by mck1117** [comment]
This is fixed both by better engine stop logic, and by overdwell protection.
